core/indexing/twelvelabs.py

```
@@ -418,6 +418,211 @@ def poll_indexed_asset_ready(
+        return None, "TwelveLabs not configured"
+    if total_size <= 0:
+        return None, "total_size must be positive"
+    try:
+        session = client.multipart_upload.create(
+            filename=filename or "video.mp4",
+            type="video",
+            total_size=int(total_size),
+        )
+        if not session.upload_id or not session.chunk_size:
+            return None, "Invalid upload session response from TwelveLabs"
+        urls: List[Dict[str, Any]] = []
```

Ruff (0.15.18)

[error] Undefined name `List`

(F821)